internal/cli/auth/register.go

```diff
@@ -15,20 +15,76 @@
 package auth
 
 import (
+	"context"
+	"errors"
 	"fmt"
+	"os"
 
 	"github.com/mongodb/mongocli/internal/cli/require"
 	"github.com/mongodb/mongocli/internal/config"
+	"github.com/pkg/browser"
 	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
 )
 
+const accountURI = "https://account.mongodb.com/account/register?fromURI=https://account.mongodb.com/account/connect"
+const govAccountURI = "https://account.mongodbgov.com/account/register?fromURI=https://account.mongodbgov.com/account/connect"
+
 type RegisterOpts struct {
 	loginOpts
```

Comment:
[nit] 2 `Run` commands, we discussed this on a previous PR, extract the parts that you really need or if you really need everything another thing can be 
```suggestion
	login loginOpts
```
so the behaviour is explicit